satellite/satellitedb/overlaycache.go

```diff
@@ -581,6 +564,12 @@ func (cache *overlaycache) UpdateAuditSuccess(ctx context.Context, nodeID storj.
 	updateFields.TotalAuditCount = dbx.Node_TotalAuditCount(totalAuditCount)
 	updateFields.AuditSuccessRatio = dbx.Node_AuditSuccessRatio(auditRatio)
 
+	if auditSuccess {
+		updateFields.LastContactSuccess = dbx.Node_LastContactSuccess(time.Now())
+	} else {
+		updateFields.LastContactFailure = dbx.Node_LastContactFailure(time.Now())
```

Comment:
if the audit is not successful, it does not necessarily mean we failed to contact the node. It just means the node failed the audit. We could have still successfully contacted the node.

Reply:
You are right. However, I've just noticed that `UpdateAudit Success` is not used anywhere. The audit reporter uses `UpdateStats` to report audit success/failure. So I am going to remove it.

Reply:
Done.